tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend_2.robot

```diff
@@ -461,4 +513,164 @@ Check amended release doesn't contain deleted subject
+    user waits until page contains accordion section   ${PUBLICATION_NAME}
+    user opens accordion section  ${PUBLICATION_NAME}
+    ${accordion}=  user gets accordion section content element  ${PUBLICATION_NAME}
+    user opens details dropdown   ${RELEASE_NAME} (Live - Latest release)  ${accordion}
+    ${details}=  user gets details content element  ${RELEASE_NAME} (Live - Latest release)  ${accordion}
+    user waits until parent contains element   ${details}   xpath:.//a[text()="View this release"]
+    user clicks button  Amend this release  ${details}
+    user clicks button  Confirm
+
+Add "upload file test filter" subject file
+    [Tags]  HappyPath
+    user clicks link  Data and files
+     user waits until page contains element  id:dataFileUploadForm-subjectTitle
+    user enters text into element  id:dataFileUploadForm-subjectTitle   New subject file footnotes
+    user chooses file   id:dataFileUploadForm-dataFile       ${FILES_DIR}upload-file-test-with-filter.csv
+    user chooses file   id:dataFileUploadForm-metadataFile   ${FILES_DIR}upload-file-test-with-filter.meta.csv
+    user clicks button  Upload data files
+    user waits until h2 is visible  Uploaded data files
+    user waits until page contains accordion section   New subject file footnotes
+    user opens accordion section   New subject file footnotes
+    ${section}=  user gets accordion section content element  New subject file footnotes
+    user checks headed table body row contains  Status           Complete  ${section}  180
+
+Add meta guidance to "New subject file" subject
+    [Tags]  HappyPath
+    user clicks link  Metadata guidance
+    user waits until h2 is visible  Public metadata guidance document
+    user enters text into element  id:metaGuidanceForm-content  Test meta guidance content
+    user waits until page contains accordion section  New subject file footnotes
+    user enters text into meta guidance data file content editor  New subject file footnotes
+    ...  meta guidance content
+    user clicks button  Save guidance
+
+Navigate to 'Footnotes' Tab
+    [Tags]  HappyPath
+    user clicks link  Footnotes
+    user waits until h2 is visible  Footnotes
+
+Add footnote to "upload file test filter" subject file
+    [Tags]  HappyPath
+    user clicks link  Create footnote
+    user waits until h2 is visible  Create footnote
+    user clicks footnote checkbox  Select all indicators and filters for this subject
+    user enters text into element  id:footnoteForm-content  upload file test filter footnote
+    user clicks button  Save footnote
+    user waits until h2 is visible  Footnotes
+
+Update "seven filters" footnote
+    [Tags]  HappyPath
+    user clicks link  Edit footnote
+    user enters text into element  id:footnoteForm-content  Updating seven filters footnote
+    user clicks button  Save footnote
+    user waits until h2 is visible  Footnotes
+
+Go to "Sign off" page and approve the amendment
```

Comment:
I'd change this to `Go to "Sign off" page for amendment` - I'm being fastidious here :)